src/conftest.py

```diff
@@ -147,6 +147,53 @@ def entity():
     }
     return entity
 
+@pytest.fixture
+def sameTypeEntitiesWithDifferentAttrs():
+    """
```

Comment:
i wonder why only testing same entity id?

Reply:
No special reason for testing with same entity id. It just happens to match my use case for this feature. Different ids would work as well although test_insert_same_type_entities_with_different_attrs would require some changes since query would return 2 records not one and utils.common.check_notifications_record requires that all entities have the same id and there is only one record.